src/prefect/tasks/prefect/flow_run.py

```diff
@@ -54,6 +57,7 @@ def run(
         parameters: dict = None,
         idempotency_key: str = None,
         new_flow_context: dict = None,
+        run_name: str = None,
```

Comment:
Since this option is available both at initialization and at run time, it should be included in the `@defaults_from_attrs` decorator on the `run` method.  Otherwise the value at initialization will never be used

Reply:
Thanks, good catch. I added this when I was looking at how this works, but then it seems to have slipped when making the commit.